### DISCUSSION TITLE
Game Anti-Cheat Detection?

### DISCUSSION THREAD

**Discussion by Sand216** [Maplespe/DWMBlurGlass#260] [Q&A] [answered]
Saw [this post](https://github.com/MicaForEveryone/MicaForEveryone/discussions/356#discussion-5607901) on the MicaForEveryone Github discussions that briefly talked about DWM hooking having a high chance to trigger anti-cheat systems in games (Second paragraph, at the end in parentheses). Is this true? If so, does it also apply to DWMBlurGlass?

This is my first post ever on Github, so let me know if I did anything incorrect. :)

**Comment by Maplespe** [comment]
I can tell you very responsibly that it won't .
China has one of the strictest anti-cheat systems in the world, and it will even scan your entire hard drive and equipment, hardware, and anywhere else, even if you don't have a so-called cheat program enabled.
But even then it still doesn't trigger any anomalies in the anti-cheat system.
Anti-cheats like Easy Anti-Cheat have even less effect.

We only inject into dwm and do not interfere with any other process.
I don't know what they are thinking, it's just possible that they don't have the technical skills to do so, they don't have any proof.

**Comment by Maplespe** [comment]
Incidentally, I have a very large number of Chinese users, I have my own platform [website](https://winmoes.com), and I also publish my own beautification software on [bilibili](https://space.bilibili.com/87195798) (a YouTube-like video platform), with over 50W total video views, and not a single instance of feedback claiming an impact on anti-cheating.

**Comment by ALTaleX531** [comment, marked as answer]
Technically, DWMBlurGlass is hard to trigger anti-cheat detection, but that doesn't rule out invasive anti-cheat software used by some games. In more cases you should be experiencing compatibility issues than getting a game ban. DWMBlurGlass will most likely not be able to inject into dwm, for example Riot Vanguard is suspected of preventing any program from injecting into dwm in order to bypass their anti-cheat detection. Of course all of this above lacks enough practical examples to support it, as no one has ever fed back on any of this.

**Reply by Sand216** [reply]
Thank you both for your replies!
